### ISSUE TITLE
hope you can update to version 1.12.2 as soon as possible

### ISSUE THREAD

**Issue by xcby-jpg** [N1nt4nd0/Xenobyte#18]
Hello, author, I hope you can update to version 1.12.2 as soon as possible. Thank you.

**Comment by xcby-jpg** [comment]
Здравствуйте, великий автор, я надеюсь, вы сможете обновить версии 1.12.2 как можно скорее.

**Comment by asc11cat** [comment]
See #17, its permanent, there will be no 'fix'.

**Comment by xcby-jpg** [comment]
No, you are wrong. China already has a lot of fixes and anti-injection plug-ins.
------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"asc11cat"<notifications@github.com&gt;;
发送时间:&nbsp;2020年3月2日(星期一) 晚上7:11
收件人:&nbsp;"N1nt4nd0/Xenobyte"<Xenobyte@noreply.github.com&gt;;
抄送:&nbsp;"ღ᭄ꦿ࿐千千结╰つ゛"<3119103572@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [N1nt4nd0/Xenobyte] hope you can update to version 1.12.2 as soon as possible (#18)




See #17, its permanent, there will be no 'fix'.

**Comment by whycantiusemyname** [comment]
> No, you are wrong. China already has a lot of fixes and anti-injection plug-ins. ------------------&nbsp;原始邮件&nbsp;------------------ 发件人:&nbsp;"asc11cat"<notifications@github.com&gt;; 发送时间:&nbsp;2020年3月2日(星期一) 晚上7:11 收件人:&nbsp;"N1nt4nd0/Xenobyte"<Xenobyte@noreply.github.com&gt;; 抄送:&nbsp;"ღ᭄ꦿ࿐千千结╰つ゛"<3119103572@qq.com&gt;;"Author"<author@noreply.github.com&gt;; 主题:&nbsp;Re: [N1nt4nd0/Xenobyte] hope you can update to version 1.12.2 as soon as possible (#18) See #17, its permanent, there will be no 'fix'. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.
国人？他意思是BUG被修复了就没有在利用的可能了
